### PR TITLE
297/circle ci

### LINKED ISSUE #297: Run tests for ipfx in cloud-based CI tool
In order to better support external users of this repository, it makes sense to set up an automated build system that is not limited to internal Allen Institute systems only.

Investigate and implement one of the start tools for this. See AllenSDK tests for example.
~200 tests; about 90% cloud-compatible.
Non-integration tests take ~10 minutes to run

### PR DESCRIPTION
**Pull request by sgratiy**
Resolves #297

### CODE REVIEW

**Comment by sgratiy** [comment]
@kschelonka I was not able to get codecov to work. Would be nice to have, but it is not explicitly called for this issue.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+      - run:
+          name: install test dependencies
+          command: |
+            . venv/bin/activate
+            pip install -r test_requirements.txt
+            pip install codecov
+
+      # run tests!
+      # this example uses Django's built-in test-runner
```
Comment (kschelonka) [inline .circleci/config.yml]: I'd remove this comment since you're using pytest to reduce confusion

**Comment by kschelonka** [comment]
Once the comment from the circleci example is removed I approve

File: .circleci/config.yml (hunk 1 above)
Comment (sgratiy) [reply .circleci/config.yml]: good catch, I have amended the commits.

**Pull request merged**